services/movie.py

```diff
@@ -0,0 +1,36 @@
+from django.db.models.query import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(genres_ids: list[int] = None,
+               actors_ids: list[int] = None) -> QuerySet[Movie]:
+    movies = Movie.objects.all()
+
+    if genres_ids:
+        movies = movies.filter(genres__id__in=genres_ids)
+    if actors_ids:
+        movies = movies.filter(actors__id__in=actors_ids)
+
+    return movies.distinct()
```

Comment:
I actually tested that with single filter for genres and there were repeating fields because of ManyToMany relations and it even retain in your further testings, you can check it:
![image](https://user-images.githubusercontent.com/107141441/225329874-2b3aa844-eafc-4c53-9f7e-fcfd8896a9cb.png)
I simply removed all repeatings as it was ordered in readme, idk what are you talking about

Reply:
Here is more simple example for you:
![image](https://user-images.githubusercontent.com/107141441/225331793-2d15284e-8873-4c5b-93a5-a1b9ee745717.png)